manipulation/src/test.py

```diff
@@ -0,0 +1,29 @@
+from subprocess import Popen, PIPE
+import sys
+
+import torch
+import onnx
+from torch.autograd import Variable
+
+def roundtrip_helper(roundtrip_binary, input_proto_path, output_proto_path):
+    rt = Popen("{} < {} > {}".format(roundtrip_binary, input_proto_path, output_proto_path), shell=True)
+    assert rt.wait() == 0
+
+    input_model = onnx.load(input_proto_path)
+    output_model = onnx.load(output_proto_path)
+
+    assert onnx.helper.printable_graph(input_model.graph) \
```

Comment:
it might be better to compare source protobufs (e.g. str(graph_a) == str(graph_b)) directly as printable_graph() doesn't print all fields.

Reply:
I think you can directly do `graph_a == graph_b`

Reply:
I can do this, but I'll have to add more code to preserve comments everywhere (almost every onnx struct has a comment field that I'm currently not preserving)